src/styles/CatalogStyle/_catalog.scss

```
@@ -0,0 +1,26 @@
+/* #region Catalog */
+main {
+  display: grid;
+  grid-template-columns: 1fr;
+  gap: 48px 46px;
+  margin: 50px 40px;
```

for tests you need to use padding so it would be part of this element 
```suggestion
  padding: 50px 40px;
```